lib/open_api_spex/plug/swagger_ui.ex

```diff
@@ -70,8 +80,8 @@ defmodule OpenApiSpex.Plug.SwaggerUI do
     <body>
     <div id="swagger-ui"></div>
 
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/swagger-ui/4.14.0/swagger-ui-bundle.js" charset="UTF-8"> </script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/swagger-ui/4.14.0/swagger-ui-standalone-preset.js" charset="UTF-8"> </script>
+    <script src="https://unpkg.com/swagger-ui-dist@#{@swagger_ui_version}/swagger-ui-bundle.js" charset="UTF-8"> </script>
+    <script src="https://unpkg.com/swagger-ui-dist@#{@swagger_ui_version}/swagger-ui-standalone-preset.js" charset="UTF-8"> </script>
```

Comment:
I don't see the need to switch from cloudflare's own cdnjs to unpkg which is also now powered by cloudflare.

Reply:
As also mentioned in the issue #627 then I could not find a way to serve latest version from cdnjs and that's why I switched to the unpkg instead. At least [https://cdnjs.cloudflare.com/ajax/libs/swagger-ui/latest/swagger-ui-bundle.js]() and [https://cdnjs.cloudflare.com/ajax/libs/swagger-ui/swagger-ui-bundle.js]() do not work and [https://cdnjs.cloudflare.com/]() does not mention any other way to reference the latest version either. And even after googling it seems that cndjs does not allow to fetch `latest` versions on purpose (see [this old issue](https://github.com/cdnjs/cdnjs/issues/360) for example among other similar issues).

If you happen to know a way to fetch latest version then there's no need to switch to unpkg indeed, otherwise there is and as mentioned in the issue then unpkg.com seems to be the _de facto_ recommendation by SwaggerUI too (not that it matters in the end really).